ClinicManagement/src/ClinicManagement.Api/Startup.cs

```diff
@@ -110,12 +110,17 @@ public void ConfigureServices(IServiceCollection services)
       services.Configure<RabbitMqConfiguration>(messagingConfig);
       services.AddSingleton<ObjectPoolProvider, DefaultObjectPoolProvider>();
       services.AddSingleton<IPooledObjectPolicy<IModel>, RabbitModelPooledObjectPolicy>();
-    }
 
-    public void ConfigureContainer(ContainerBuilder builder)
-    {
-      bool isDevelopment = (_env.EnvironmentName == "Development");
-      builder.RegisterModule(new DefaultInfrastructureModule(isDevelopment, Assembly.GetExecutingAssembly()));
+      var assemblies = new Assembly[]
+      {
+        typeof(Room).Assembly,
+        typeof(DefaultInfrastructureModule).Assembly,
+        typeof(Startup).Assembly,
+      };
+      services.AddMediatR(config => config.RegisterServicesFromAssemblies(assemblies));
+
```

Comment:
To keep things consistent between projects, MediatR is registered on the Startup class too. The same approach used in the FrontDesk project